azure-iot-hub/tests/common_fixtures.py

```diff
@@ -0,0 +1,47 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from azure.iot.hub.iothub_registry_manager import IoTHubRegistryManager
+
+"""---Constants---"""
+
+fake_shared_access_key = "Zm9vYmFy"
+fake_shared_access_key_name = "alohomora"
+
+fake_primary_key = "petrificus"
+fake_secondary_key = "totalus"
+fake_primary_thumbprint = "HELFKCPOXAIR9PVNOA3"
+fake_secondary_thumbprint = "RGSHARLU4VYYFENINUF"
+fake_hostname = "beauxbatons.academy-net"
+fake_device_id = "MyPensieve"
+fake_module_id = "Divination"
+fake_managed_by = "Hogwarts"
+fake_etag = "taggedbymisnitryofmagic"
+fake_status = "flying"
+
+
+"""----Shared fixtures----"""
+
+
+@pytest.fixture(scope="function")
+def mock_service_operations(mocker):
+    mock_service_operations_init = mocker.patch(
+        "azure.iot.hub.protocol.iot_hub_gateway_service_ap_is20190630.ServiceOperations"
+    )
+    return mock_service_operations_init.return_value
+
+
+@pytest.fixture(scope="function")
+def iothub_registry_manager():
```

Comment:
Does this need to be a shared fixture?

Reply:
i guess everything can be moved inside since it is all one file

---
In reply to: [332237109](https://github.com/Azure/azure-iot-sdk-python/pull/340#discussion_r332237109) [](ancestors = 332237109)